static-analysis/teammates-pmd.xml

```diff
@@ -11,143 +11,174 @@
-    <rule ref="category/java/errorprone.xml/ReturnEmptyArrayRatherThanNull"/>
-    <rule ref="category/java/errorprone.xml/ReturnFromFinallyBlock"/>
-    <rule ref="category/java/errorprone.xml/SingleMethodSingleton"/>
-    <rule ref="category/java/errorprone.xml/SingletonClassReturningNewInstance"/>
-    <rule ref="category/java/errorprone.xml/StringBufferInstantiationWithChar"/>
-    <rule ref="category/java/errorprone.xml/TestClassWithoutTestCases"/>
-    <rule ref="category/java/errorprone.xml/UnconditionalIfStatement"/>
-    <rule ref="category/java/errorprone.xml/UnnecessaryBooleanAssertion"/>
-    <rule ref="category/java/errorprone.xml/UnnecessaryCaseChange"/>
-    <rule ref="category/java/errorprone.xml/UnnecessaryConversionTemporary"/>
-    <rule ref="category/java/errorprone.xml/UnusedNullCheckInEquals"/>
-    <rule ref="category/java/errorprone.xml/UseEqualsToCompareStrings"/>
-    <rule ref="category/java/errorprone.xml/UselessOperationOnImmutable"/>
-
-    <rule ref="category/java/multithreading.xml/AvoidThreadGroup"/>
-    <rule ref="category/java/multithreading.xml/DontCallThreadRun"/>
-    <rule ref="category/java/multithreading.xml/DoubleCheckedLocking"/>
-    <rule ref="category/java/multithreading.xml/UnsynchronizedStaticDateFormatter"/>
-    <rule ref="category/java/multithreading.xml/UseNotifyAllInsteadOfNotify"/>
-
-    <rule ref="category/java/performance.xml/AddEmptyString"/>
-    <rule ref="category/java/performance.xml/AppendCharacterWithChar"/>
-    <rule ref="category/java/performance.xml/AvoidArrayLoops"/>
-    <rule ref="category/java/performance.xml/BigIntegerInstantiation"/>
-    <rule ref="category/java/performance.xml/BooleanInstantiation"/>
-    <rule ref="category/java/performance.xml/ConsecutiveAppendsShouldReuse"/>
-    <rule ref="category/java/performance.xml/ConsecutiveLiteralAppends"/>
-    <rule ref="category/java/performance.xml/InefficientStringBuffering"/>
-    <rule ref="category/java/performance.xml/InsufficientStringBufferDeclaration"/>
-    <rule ref="category/java/performance.xml/OptimizableToArrayCall"/>
-    <rule ref="category/java/performance.xml/RedundantFieldInitializer"/>
-    <rule ref="category/java/performance.xml/StringInstantiation"/>
-    <rule ref="category/java/performance.xml/StringToString"/>
-    <rule ref="category/java/performance.xml/TooFewBranchesForASwitchStatement"/>
-    <rule ref="category/java/performance.xml/UnnecessaryWrapperObjectCreation"/>
-    <rule ref="category/java/performance.xml/UseArrayListInsteadOfVector"/>
-    <rule ref="category/java/performance.xml/UseArraysAsList"/>
-    <rule ref="category/java/performance.xml/UseIndexOfChar"/>
-    <rule ref="category/java/performance.xml/UselessStringValueOf"/>
-    <rule ref="category/java/performance.xml/UseStringBufferForStringAppends"/>
-    <rule ref="category/java/performance.xml/UseStringBufferLength"/>
+    <rule ref="category/java/errorprone.xml">
+        <!-- To enforce if there is opportunity to -->
```

Comment:
Should the `may consider enforcing` and `To enforce if there is opportunity to` be a separate  issue?

Reply:
I'll standardize to the first variant.